frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/SchemaGenerationErrorsPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import classes from './SchemaGenerationErrorsPanel.module.css';
+import React from 'react';
+import { Alert, Button, ErrorMessage, Paragraph } from '@digdir/design-system-react';
+import { Trans, useTranslation } from 'react-i18next';
+import { XMarkIcon } from '@navikt/aksel-icons';
+
+export interface SchemaGenerationErrorsPanelProps {
+  onCloseErrorsPanel: () => void;
+  schemaGenerationErrorMessages: string[];
+}
+
+export const SchemaGenerationErrorsPanel = ({
+  onCloseErrorsPanel,
+  schemaGenerationErrorMessages,
+}: SchemaGenerationErrorsPanelProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Alert severity='danger'>
+      <div className={classes.errorPanel}>
+        <div>
+          <Paragraph>{t('api_errors.DM_01')}</Paragraph>
+          <ul>
+            {schemaGenerationErrorMessages?.map((errorMessage, index) => {
+              return (
+                <li key={index}>
+                  <ErrorMessage>
+                    {errorMessage.includes(
+                      'member names cannot be the same as their enclosing type',
+                    ) ? (
+                      <Trans
```

Comment:
What do you think about using the `t` instead of the trans method? I think we could do something like the following code: 

`t('api_errors.DM_CsharpCompiler_NameCollision', {
    nodeName: extractNodeNameFromError(errorMessage)
})`

Reply:
That exact langkey is for the Norwegian mapping tho, so it is not the same text 🫣
But you might mean that we should add a new text-key? It feels a bit wierd to me since this text must be in English, so it is not actually a key used for i18n, I think? 🤔
But I might add it as a const in the same file instead? So it is easier to understand the origin of the particular text.

Maybe something like: `KNOWN_API_ERROR_MESSAGE_CSHARP_COMPILER_NAME_COLLISION` 
or only: `API_ERROR_MESSAGE_COMPILER_NAME_COLLISION`?